### ISSUE TITLE
389-ds-base-2.0.14-1.el9 rpminspect: chdir: No such file or directory

### ISSUE THREAD

**Issue by jimbair** [rpminspect/rpminspect#776]
After running tag-runner against c9s-pending yesterday, I ran into the following error with 389-ds-base:

```
[jbair@awvr rpminspect-tag-runner]$  grep 'chdir' logs/*stderr | sort | uniq -c
   1320 logs/389-ds-base-2.0.14-1.el9.log.stderr:rpminspect: chdir: No such file or directory
[jbair@awvr rpminspect-tag-runner]$ 

```

The command ran:
`rpminspect-centos -s VERIFY -w mytmp -o logs/389-ds-base-2.0.14-1.el9.log 389-ds-base-2.0.14-1.el9`

I know these can sometimes be fiddly to track, but if it helps, this is also one of the first builds run as tag-runner fired up in alphabetical order, so if you're having trouble reproducing, let me know and I can try to help.

**Comment by dcantrell** [comment]
I can't reproduce this locally.  I'm just doing:

```
rpminspect-centos -v -s VERIFY -o results.log 389-ds-base-2.0.14-1.el9
```

And everything passes and the exit code is 0.  There is no results.log file.

**Comment by jimbair** [comment]
Yeah, I re-ran the same test on the same version and it ran without issue:

```
[jbair@awvr ~]$ mkdir test
[jbair@awvr ~]$ cd test
[jbair@awvr test]$ ls
[jbair@awvr test]$ mkdir mytmp
[jbair@awvr test]$ mkdir logs
[jbair@awvr test]$ rpminspect-centos -s VERIFY -w mytmp -o logs/389-ds-base-2.0.14-1.el9.log 389-ds-base-2.0.14-1.el9
[jbair@awvr test]$ echo $?
0
[jbair@awvr test]$ 
```

I think this has to do with threading + user defined tmp dir if I had to guess. Let's keep this open for now and I'll try to capture it on my next c9s run and see what state things are in, and if not, I can close it out until we can easily repeat it.
